app/helpers/application_helper.rb

```diff
@@ -1214,7 +1215,7 @@ def start_page_allowed?(start_page)
   end
 
   def allowed_filter_db?(db)
-    return false if db.start_with?('Container') && !get_vmdb_config[:product][:containers]
+    return false if (db.nil? || db.start_with?('Container')) && !get_vmdb_config[:product][:containers]
     true
```

Comment:
@abonas : here we go with my initial comment. No **bloody chance** anyone revisiting this code any time later will gues anywhere near this explanation. This should be nicely formatted method with a spec explaining the case or at least a comment..